shared-libs/transitions/src/transitions/registration.js

```diff
@@ -40,6 +41,23 @@ const getNameField = (params, prefix) => {
   return defaultNameField;
 };
 
+const getPatientPhoneField = (settings, currentForm) => {
+  // Get the current form for which we are running transition form
+  if(settings && currentForm && settings.forms) {
+    const formDef = settings.forms[currentForm];
+
+    if (formDef && formDef.fields) { 
+      // Get the phone field i.e.the field with type phone_number in the form
+      const phoneField = (Object.keys(formDef.fields).filter(key => formDef.fields[key].type === 'phone_number'));
+      // Return the phone field
+      if (phoneField && phoneField[0]) {
+        return phoneField[0];
+      }
+    }  
+  }
+};
+
```

Comment:
Please consider early returns in this function to reduce depth, like: 
```suggestion
const getPatientPhoneField = (currentForm) => {
  const formDefinition = utils.getForm(currentForm);
  if (!formDefinition) {
    return;
  }
  
  const phoneField = Object.keys(formDef.fields).find(key => formDef.fields[key].type === 'phone_number');
  return phoneField; 
};
```

I'm also pretty sure we should have a way to get a specific form config, instead of having to go through all settings. 
Found it: https://github.com/medic/cht-core/blob/0e4c653890ce020064f7f0a07fa83c89ae7d06cf/shared-libs/transitions/src/lib/utils.js#L162